runtime/onert/core/src/backend/builtin/train/TensorRegistry.h

```diff
@@ -18,6 +18,7 @@
 #define __ONERT_BACKEND_BUILTIN_TRAIN_TENSOR_REGISTRY_H__
 
 #include <backend/train/ITensorRegistry.h>
+#include <backend/train/ITrainableTensor.h>
```

Comment:
@ragmani It requires `ITrainableTensor`. May I ask why you want to remove this inclusion?

Reply:
This header file needs `ITrainableTensor` because of the virtual function of `ITensorRegistry` that is base class. When 
`ITrainableTensor` dependency is removed from `ITensorRegistry`, `ITrainableTensor` dependency disappears naturally in this file. So I think this "include" can be removed.
Anyway it's optional. You can ignore my suggestion above.

Reply:
Now, I understood your point. I will remove it.